local-cli/bundle/bundle.js

```diff
@@ -8,26 +8,30 @@
  */
 
 const buildBundle = require('./buildBundle');
-const bundleCommandLineArgs = require('./bundleCommandLineArgs');
-const parseCommandLine = require('../util/parseCommandLine');
 const outputBundle = require('./output/bundle');
 const outputPrepack = require('./output/prepack');
+const bundleCommandLineArgs = require('./bundleCommandLineArgs');
 
 /**
  * Builds the bundle starting to look for dependencies at the given entry path.
  */
-function bundleWithOutput(argv, config, output, packagerInstance) {
-  const args = parseCommandLine(bundleCommandLineArgs, argv);
+function bundleWithOutput(argv, config, args, output, packagerInstance) {
   if (!output) {
     output = args.prepack ? outputPrepack : outputBundle;
   }
   return buildBundle(args, config, output, packagerInstance);
-
 }
 
-function bundle(argv, config, packagerInstance) {
```

Comment:
For the sake of easier merge can you provide a shim for the old interface?
args here changed signature, our internal script expects this one:
```
--entry-file,/path/SomeAppBundle.js,--platform,ios,--dev,true,--bundle-output,/path/buck-out/gen/__SomeAppJS#dev_js__/main.jsbundle,--assets-dest,/path/buck-out/gen/Libraries/SomeApp#dev_res__,--sourcemap-output,/path/buck-out/gen/Libraries/SomeAppJS#dev_source_map__/source.map
```